src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs

```diff
@@ -200,7 +198,7 @@ void HandleDrop(object sender, Microsoft.UI.Xaml.DragEventArgs e)
 				element = ve;
 			}
 
-			var args = new DropEventArgs(datapackage?.View);
+			var args = new DropEventArgs(datapackage?.View!, (relativeTo) => GetPosition(relativeTo, e));
```

Comment:
if the `data package?.View` is null maybe we just exit early so we can avoid the null forgiveness operator?